doc/Getting-Started-MacOSX.md

```diff
@@ -36,8 +36,8 @@ Now we want to get EF for dotnet.
 
 Go here: https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dotnet and see Installing the Tools section. 
 
-Edit the Web.Host project file (right click project name and there is an Tools->Edit File option) and add following:  
-	<ItemGroup> <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" /> </ItemGroup>
+Edit the Web.Host project file (right click project name and there is an Tools->Edit File option) and add following:
+	~~~~<ItemGroup> <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" /> </ItemGroup>~~~~
```

Comment:
Still not shown. I think, entering a new line (before code block) instead of adding ~~~~ can fix problem.